prusti-server/src/server_message.rs

```diff
@@ -0,0 +1,22 @@
+// © 2023, ETH Zurich
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use viper::VerificationResult;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub enum ServerMessage {
```

Comment:
Can you add a comment explaining what this is, where it comes from? Each variant should also have a brief explanation.